vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -303,7 +303,7 @@ if (-not ($validContainerRuntimes -contains $containerRuntime)) {
 }
 
 $windowsSKU = $env:WindowsSKU
-$validSKU = @('2019', '2004')
+$validSKU = @('2019', '2019-containerd', '2004')
```

Comment:
I think that we can remove 2004 now.

Reply:
We can do the removal work in another PR.

Reply:
Also, 2004 related code exists in several parts, like the base image, etc. please let's keep it until we bring 2004 back into production.

Reply:
2004 will not be back but we can keep the code until a new version comes.